PowerEditor/src/Notepad_plus.cpp

```diff
@@ -5937,10 +5939,6 @@ void Notepad_plus::prepareBufferChangedDialog(Buffer * buffer)
 
 void Notepad_plus::notifyBufferChanged(Buffer * buffer, int mask)
 {
-	// To avoid to crash while MS-DOS style is set as default language,
-	// Checking the validity of current instance is necessary.
-	if (!this) return;

```

Comment:
`this` should never be null. Some compiler even ignore comparing `this` with `NULL`.
If `this` is null then something bad has happened leading  to undefined behavior.

@donho 
I am getting crash even with this check.